frontend/src/routes/login.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="ts">
+  let username = ""
+  let password = ""
+
+  const signIn = async () => {
+    let loginStatus = ""
+    const payload = `username=${username}&password=${password}`
+
+    await fetch("http://localhost:2000/login/", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/x-www-form-urlencoded",
+      },
+      body: payload,
+    })
+      .then((response) => response.json())
+      .then((data) => (loginStatus = data.status))
+
+    if (loginStatus === "success") {
+      alert("hooray you signed in")
```

Comment:
We should probably do something with window.location or window.history and redirect the user somewhere
```suggestion
    if (loginStatus === "success") {
      alert("hooray you signed in")
      window.location.href = "/"
```

Reply:
Yeah obviously the alert is also just a temp as we dont have anywhere to redirect really :P